conf/modules.config

```diff
@@ -58,7 +58,7 @@ process {
         ext.args2 = '--no-same-owner'
     }
 
-    withName: 'UNTAR_.*|STAR_GENOMEGENERATE|HISAT2_BUILD' {
+    withName: 'UNTAR_.*|STAR_GENOMEGENERATE|STAR_GENOMEGENERATE_IGENOMES|HISAT2_BUILD' {
```

Comment:
@mahesh-panchal if you are around can you double-check this logic looks good. I have tested and it seems to be working but wanted a sanity check.